lib/src/client.dart

```diff
@@ -28,10 +27,7 @@ abstract class Client {
   /// Currently this will create an [IOClient] if `dart:io` is available and
   /// throw an [UnsupportedError] otherwise. In the future, it will create a
   /// [BrowserClient] if `dart:html` is available.
```

Comment:
This comment isn't relevant anymore, since you can't import this library without "dart:io" being available, can you?